DependencyInjection/CmfSeoExtension.php

```diff
@@ -59,9 +59,9 @@ public function load(array $configs, ContainerBuilder $container)
 
         $sonataBundles = array();
         if ($this->isConfigEnabled($container, $config['persistence']['phpcr'])) {
-            $container->setParameter($this->getAlias().'.backend_type_phpcr', true);
+            $container->setParameter('cmf_seo.backend_type_phpcr', true);
```

Comment:
Why was that change needed? Thought that is the common way to use the bundle's alias.